### DISCUSSION TITLE
貢獻開源專案其實很簡單

### DISCUSSION THREAD

**Discussion by gnsJhenJie** [1101-GS4719-C/Python-Community-GS4719-C#574] [閒聊八卦]
GitHub上有許多的開源(Open Source)專案，這些專案大部分開放讓大家一起貢獻專案內容(像是新增功能, 除錯...)，但其實也不見得要很會寫程式才能對這些專案做出貢獻。  

最近看了Google一篇論文附上的[Code](https://github.com/google-research/vision_transformer)，但發現其中一個檔案的超連結點進去竟然是404 Not found，於是我就把連結更正以後，發了一個Pull Request給Google，過幾天PR被接受我就變成Google專案的貢獻者之一了～   

<img width="1125" alt="Screen Shot 2022-04-27 at 21 06 02" src="https://user-images.githubusercontent.com/32983347/165524626-ce6db397-a6f7-4124-b635-0cd14ac440d2.png">
（可以看到就真的只是改幾個字這麼輕鬆）  

<br>
<br>

除了可以在專案的Contributor中看到自己外，自己GitHub的頁面也會出現為Google-research貢獻過的紀錄   
<img width="282" alt="Screen Shot 2022-04-27 at 21 01 56" src="https://user-images.githubusercontent.com/32983347/165523909-a8429f9f-5853-4edc-b974-b96d665e4a38.png">
<img width="716" alt="Screen Shot 2022-04-27 at 21 02 57" src="https://user-images.githubusercontent.com/32983347/165524094-6fa7a1f6-6dbc-44f2-a839-c51f5e766049.png">
（看起來很厲害，但實際上就是更新個連結而已 XD）

**Comment by jason79461385** [comment]
看起來很讚，這樣自己的主頁看起來也更有水準！不會都是自己廢廢的專案😥😥

**Comment by Lynn12378** [comment]
看起來超強的XDD，但是一般人也不會去關注這些消息，平常都有跟進新專案才有這個機會，樓主肯定是電神。
